node/src/web.rs

```diff
@@ -114,31 +114,46 @@ async fn third_party_licenses() -> Html<&'static str> {
 #[derive(Clone, Serialize)]
 pub struct StaticWebData {
     pub near_signer_public_key: near_crypto::PublicKey,
+    pub near_p2p_public_key: near_crypto::PublicKey,
     pub near_responder_public_keys: Vec<near_crypto::PublicKey>,
     pub tee_participant_info: Option<TeeParticipantInfo>,
 }
 
 fn get_public_keys(
     secrets_config: &SecretsConfig,
-) -> (near_crypto::PublicKey, Vec<near_crypto::PublicKey>) {
+) -> (
+    near_crypto::PublicKey,
+    near_crypto::PublicKey,
+    Vec<near_crypto::PublicKey>,
+) {
```

Comment:
```suggestion
) -> PublicKeys {
```

I think a struct would be better to return here over a tuple. It will make it less likely we mix up which key is what.
```rust
PublicKeys {
    near_signer_public_key: near_crypto::PublicKey,
    near_p2p_public_key: near_crypto::PublicKey,
    near_responder_public_keys: Vec<near_crypto::PublicKey>,
}
```